### ISSUE TITLE
Linux : Shovel Knight (HDD version) will not launch when eboot.bin is passed on command line

### ISSUE THREAD

**Issue by 7oxicshadow** [RPCS3/rpcs3#4803]
I use 3rd party launchers for my games so I always load any games via command line. Passing an eboot.bin from a CD game launches fine an works as expected.

I have shovel knight installed to the virtual PS3 hdd and if you pass the eboot.bin from the ps3 hdd location as a boot parameter on the command line it starts to boot and then fails with a 

`"F {PPU[0x100000b] Thread (Achievements) [0x002bbedc]} St13runtime_error thrown: Trap! (0x2bbedc)"`

If I then re-launch the game within the UI that remains open after the crash it loads fine and plays as expected. I am not sure, but it is as if it tries to launch before all the modules have loaded?

I am using appimage build "0.0.5-6956-af71cf092"

**Comment by AniLeo** [comment]
Is this still an issue? Could have been solved with cache system changes, please upload RPCS3.log after you get that weird Trap!

**Comment by 7oxicshadow** [comment]
Still the same. Works fine from GUI but not from command line:
```
RPCS3 v0.0.6-7932-d7ad991b Alpha | HEAD
Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz | 8 Threads | 31.36 GiB RAM | AVX+ | TSX
E HLE: evdev device 0 reconnected
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-6Yyxc1KuTSdQPc2pVKocTN-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-gSKMd9JMc8hV6axMtUp0ab-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-sRT1TF7shTc7X4ift8c3g8-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-kaArzr5x8NwcpwvYWHhTcs-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-f7XzMKJ5ib1ccveViURwjH-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-kVGW4NkPy7g4EUe2umzgSf-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-cbAPX6FTT9mME7eufWAHZA-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-9e8eTFu1bL0E8o8EJjyzo7-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-PeoXJXn41ejVgyNJJE6StR-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-cuCmLuAM9vNXkZmAizy46g-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-c8HkJMGYsZh9K9pKebMfZv-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-TVWgTyLq1GS6YRgAeHj4Ne-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-PQUSCohd8qpZ0aonZgpa18-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-YdtgFkVzawphHdcSjsbXJy-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-EcYx6MLwHj1q0tQF8dKUfm-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-iMQZddZjcaYrKPfhFLJVHw-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-kaqwt5Qkpx3kxgshzuiSjw-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-jYw8Y3dnPMkCWMRKXVtWW2-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-PqaQYRzunMvgdWdffvwbzL-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-tbth5Mf7FJgE3zYNycsTrS-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-Gtpe1ewydtcNLpnu10dvdp-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-RPWdH6SRiYJnupCXt686gu-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-Ncvyzrg05MsULaTHmwQia4-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-2nb9AK7kgvtbS91r5tRqps-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-Q9qnuhE5856hAb7kfpswk1-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-iuynjCwuEJH6zmo6HqfRp6-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-pNjAVGnsYejK9StbcYZSjX-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-7q0VtWdtSTmrFSJv7aqJWp-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-bwH7AWSp1vXimTAhc9Wvz9-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-mnEGvw5w1cbUuJKoqCcPzS-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-FS1GdvMb1EyYCU0L5kXvM3-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-k7ZXjTy5uXUVzApcwZVyV1-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-NzP9yE26WcpYXuXmnxG0wN-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-eHxnvev6XrzrGXjxi7yrKw-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-20fGEitep8YgXiwF9PCssR-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-0CXLXWA0Nm8H2ydAcGkF3Q-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-qKuWW9ck1dSYPnEhfpEvJE-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-noVzKM0eWPm3AUZLpt5Cb4-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-n2jbHFs6h1eHrk0jx1Ekj2-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-3znseSMRAogZAFppt75iVq-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-ESC8GMt3C6aeambNkmmhjx-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-kecjCuiysb36Gc5QL1ZbQu-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-YLgirczJ2TvX56KSzGhjxs-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-rUFLWMM5FPvegniKcuVQJb-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-Gm2w6Hb0uin7RFv9c5eqgK-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-W44T0ok5fnpeN81MmA7xMM-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-pT9J6W0kGGYCT7aX38ZMHm-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-KCXPzQvyKS0TkKaAdvtP8r-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} SPU: SPU Recompiler Runtime initialized...
S {PPU[0x1000000] Thread (main_thread)} SPU: SPU Runtime: Built 162 functions.
U {PPU[0x1000000] Thread (main_thread) [0x006efe64]} sys_prx TODO: _sys_prx_register_module()
U {PPU[0x1000000] Thread (main_thread) [0x006f00bc]} sys_prx TODO: _sys_prx_register_library(library=*0x6f26a0)
S {PPU[0x1000000] Thread (main_thread) [0x006f09ac]} PPU: LLVM: Loaded module v1-tane-1z3Pyu8YVnnorJiQ2hfu1e-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread) [0x006f09ac]} sys_prx: Loaded module: /dev_flash/sys/external/libsysmodule.sprx
U {PPU[0x1000000] Thread (main_thread) [0x00733d8c]} HLE TODO: Unimplemented syscall sys_ss_access_control_engine -> CELL_OK
S {PPU[0x1000000] Thread (main_thread) [0x006f14d0]} PPU: LLVM: Loaded module v1-tane-XCgQWMc1s99n1WTwrpWvw4-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread) [0x006f14d0]} sys_prx: Loaded module: /dev_flash/sys/external/libgcm_sys.sprx
S {PPU[0x1000000] Thread (main_thread) [0x006f14d0]} PPU: LLVM: Loaded module v1-tane-TP3c8YsdNggjaG5WRjcboe-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread) [0x006f14d0]} PPU: LLVM: Loaded module v1-tane-LWAKW9c35vXJad6uHkwzhK-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread) [0x006f14d0]} sys_prx: Loaded module: /dev_flash/sys/external/libsre.sprx
S {PPU[0x1000000] Thread (main_thread) [0x006f14d0]} PPU: LLVM: Loaded module v1-tane-fLV79QqHgPi90UziTsWrHo-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread) [0x006f14d0]} sys_prx: Loaded module: /dev_flash/sys/external/libfs.sprx
U {PPU[0x1000000] Thread (main_thread) [0x006f061c]} sys_prx TODO: _sys_prx_register_module()
U {PPU[0x1000001] Thread (Game Thread) [0x006f1038]} sys_prx TODO: _sys_prx_unload_module(id=0x23003e00, flags=0x0, pOpt=*0x0)
U {PPU[0x1000001] Thread (Game Thread) [0x002ed27c]} sceNp TODO: sceNpBasicRegisterHandler
E {PPU[0x1000002] Thread (fios mediathread 2) [0x007b2090]} 'sys_fs_open' failed with 0x80010006 : CELL_ENOENT, “/dev_hdd0/game/NPUB31682/USRDIR/data/global.pak” [1]
E {PPU[0x1000006] Thread (Save Thread) [0x002f55c8]} 'cellSaveDataAutoLoad2' failed with 0x8002b401 : CELL_SAVEDATA_ERROR_CBRESULT [1]
E {PPU[0x100000b] Thread (Achievements) [0x002b5440]} 'sceNpTrophyCreateContext' failed with 0x8002291b : SCE_NP_TROPHY_ERROR_CONF_DOES_NOT_EXIST [1]
S {PPU[0x1000001] Thread (Game Thread) [0x006f09ac]} PPU: LLVM: Loaded module v1-tane-2my7r7Nd2rncZdtcuT9g60-000007-skylake.obj
-------> F {PPU[0x100000b] Thread (Achievements) [0x002bbedc]} St13runtime_error thrown: Trap! (0x2bbedc) <---------
S {PPU[0x1000001] Thread (Game Thread) [0x006f09ac]} PPU: LLVM: Loaded module v1-tane-aaAKL9HJRRAWFrQ1t0vFNk-000007-skylake.obj
S {PPU[0x1000001] Thread (Game Thread) [0x006f09ac]} sys_prx: Loaded module: /dev_flash/sys/external/libnet.sprx
```

**Comment by 7oxicshadow** [comment]
For comparison. This is what you get when loading from the GUI:

```
E HLE: evdev device 0 reconnected
S LDR: Boot successful.
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-6Yyxc1KuTSdQPc2pVKocTN-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-gSKMd9JMc8hV6axMtUp0ab-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-sRT1TF7shTc7X4ift8c3g8-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-kaArzr5x8NwcpwvYWHhTcs-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-f7XzMKJ5ib1ccveViURwjH-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-kVGW4NkPy7g4EUe2umzgSf-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-cbAPX6FTT9mME7eufWAHZA-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-9e8eTFu1bL0E8o8EJjyzo7-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-PeoXJXn41ejVgyNJJE6StR-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-cuCmLuAM9vNXkZmAizy46g-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-c8HkJMGYsZh9K9pKebMfZv-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-TVWgTyLq1GS6YRgAeHj4Ne-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-PQUSCohd8qpZ0aonZgpa18-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-YdtgFkVzawphHdcSjsbXJy-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-EcYx6MLwHj1q0tQF8dKUfm-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-iMQZddZjcaYrKPfhFLJVHw-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-kaqwt5Qkpx3kxgshzuiSjw-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-jYw8Y3dnPMkCWMRKXVtWW2-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-PqaQYRzunMvgdWdffvwbzL-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-tbth5Mf7FJgE3zYNycsTrS-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-Gtpe1ewydtcNLpnu10dvdp-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-RPWdH6SRiYJnupCXt686gu-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-Ncvyzrg05MsULaTHmwQia4-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-2nb9AK7kgvtbS91r5tRqps-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-Q9qnuhE5856hAb7kfpswk1-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-iuynjCwuEJH6zmo6HqfRp6-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-pNjAVGnsYejK9StbcYZSjX-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-7q0VtWdtSTmrFSJv7aqJWp-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-bwH7AWSp1vXimTAhc9Wvz9-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-mnEGvw5w1cbUuJKoqCcPzS-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-FS1GdvMb1EyYCU0L5kXvM3-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-k7ZXjTy5uXUVzApcwZVyV1-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-NzP9yE26WcpYXuXmnxG0wN-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-eHxnvev6XrzrGXjxi7yrKw-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-20fGEitep8YgXiwF9PCssR-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-0CXLXWA0Nm8H2ydAcGkF3Q-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-qKuWW9ck1dSYPnEhfpEvJE-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-noVzKM0eWPm3AUZLpt5Cb4-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-n2jbHFs6h1eHrk0jx1Ekj2-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-3znseSMRAogZAFppt75iVq-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-ESC8GMt3C6aeambNkmmhjx-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-kecjCuiysb36Gc5QL1ZbQu-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-YLgirczJ2TvX56KSzGhjxs-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-rUFLWMM5FPvegniKcuVQJb-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-Gm2w6Hb0uin7RFv9c5eqgK-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-W44T0ok5fnpeN81MmA7xMM-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-pT9J6W0kGGYCT7aX38ZMHm-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} PPU: LLVM: Loaded module v1-tane-KCXPzQvyKS0TkKaAdvtP8r-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread)} SPU: SPU Recompiler Runtime initialized...
S {PPU[0x1000000] Thread (main_thread)} SPU: SPU Runtime: Built 315 functions.
U {PPU[0x1000000] Thread (main_thread) [0x006efe64]} sys_prx TODO: _sys_prx_register_module()
U {PPU[0x1000000] Thread (main_thread) [0x006f00bc]} sys_prx TODO: _sys_prx_register_library(library=*0x6f26a0)
S {PPU[0x1000000] Thread (main_thread) [0x006f09ac]} PPU: LLVM: Loaded module v1-tane-1z3Pyu8YVnnorJiQ2hfu1e-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread) [0x006f09ac]} sys_prx: Loaded module: /dev_flash/sys/external/libsysmodule.sprx
U {PPU[0x1000000] Thread (main_thread) [0x00733d8c]} HLE TODO: Unimplemented syscall sys_ss_access_control_engine -> CELL_OK
S {PPU[0x1000000] Thread (main_thread) [0x006f14d0]} PPU: LLVM: Loaded module v1-tane-XCgQWMc1s99n1WTwrpWvw4-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread) [0x006f14d0]} sys_prx: Loaded module: /dev_flash/sys/external/libgcm_sys.sprx
S {PPU[0x1000000] Thread (main_thread) [0x006f14d0]} PPU: LLVM: Loaded module v1-tane-TP3c8YsdNggjaG5WRjcboe-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread) [0x006f14d0]} PPU: LLVM: Loaded module v1-tane-LWAKW9c35vXJad6uHkwzhK-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread) [0x006f14d0]} sys_prx: Loaded module: /dev_flash/sys/external/libsre.sprx
S {PPU[0x1000000] Thread (main_thread) [0x006f14d0]} PPU: LLVM: Loaded module v1-tane-fLV79QqHgPi90UziTsWrHo-000007-skylake.obj
S {PPU[0x1000000] Thread (main_thread) [0x006f14d0]} sys_prx: Loaded module: /dev_flash/sys/external/libfs.sprx
U {PPU[0x1000000] Thread (main_thread) [0x006f061c]} sys_prx TODO: _sys_prx_register_module()
U {PPU[0x1000001] Thread (Game Thread) [0x006f1038]} sys_prx TODO: _sys_prx_unload_module(id=0x23003e00, flags=0x0, pOpt=*0x0)
U {PPU[0x1000001] Thread (Game Thread) [0x002ed27c]} sceNp TODO: sceNpBasicRegisterHandler
S RSX: New program compiled successfully x4
E {PPU[0x1000002] Thread (fios mediathread 2) [0x007b2090]} 'sys_fs_open' failed with 0x80010006 : CELL_ENOENT, “/dev_hdd0/game/NPUB31682/USRDIR/data/global.pak” [1]
E {PPU[0x1000006] Thread (Save Thread) [0x002f55c8]} 'cellSaveDataAutoLoad2' failed with 0x8002b401 : CELL_SAVEDATA_ERROR_CBRESULT [1]
E {PPU[0x1000006] Thread (Save Thread) [0x002c7ed0]} 'cellSaveDataAutoLoad2' failed with 0x8002b401 : CELL_SAVEDATA_ERROR_CBRESULT [2]
E {PPU[0x100000b] Thread (Achievements) [0x002b552c]} sceNpTrophy: sceNpTrophyRegisterContext(context=0x1, handle=0x1, statusCb=*0x53c128, arg=*0x0, options=0x0)
S {PPU[0x1000001] Thread (Game Thread) [0x006f09ac]} PPU: LLVM: Loaded module v1-tane-2my7r7Nd2rncZdtcuT9g60-000007-skylake.obj
S {PPU[0x1000001] Thread (Game Thread) [0x006f09ac]} PPU: LLVM: Loaded module v1-tane-aaAKL9HJRRAWFrQ1t0vFNk-000007-skylake.obj
S {PPU[0x1000001] Thread (Game Thread) [0x006f09ac]} sys_prx: Loaded module: /dev_flash/sys/external/libnet.sprx
E {PPU[0x1000001] Thread (Game Thread) [0x002c1568]} 'cellPadGetData' failed with 0x80121107 : CELL_PAD_ERROR_NO_DEVICE [1]
E {PPU[0x1000001] Thread (Game Thread) [0x002c1568]} 'cellPadGetData' failed with 0x80121107 : CELL_PAD_ERROR_NO_DEVICE [2]
E {PPU[0x1000001] Thread (Game Thread) [0x002c1568]} 'cellPadGetData' failed with 0x80121107 : CELL_PAD_ERROR_NO_DEVICE [3]
E {cellAudio Thread} cellAudio: Could not enqueue buffer onto audio backend. Attempting to recover... x5```

**Comment by digitaldude555** [comment]
@7oxicshadow update?

**Comment by AniLeo** [comment]
Still misses RPCS3.log too, just has copy paste from the emulator

**Comment by RainbowCookie32** [comment]
Works fine here. Launched from the terminal on Manjaro

![image](https://user-images.githubusercontent.com/16805474/75096256-9efe6800-557c-11ea-9dbb-aafd11318da9.png)

**Comment by 7oxicshadow** [comment]
Confirmed. It will successfully launch from command line now.

As a follow up question. I have the game patched to 1.02 but when I launch from the command line it launches the unpatched version. Does a patch typically have a different eboot somewhere else on the PS3 virtual drive?

**Comment by digitaldude555** [comment]
@7oxicshadow  you can close your issue if its fixed...
